src/Common/Handlers/InfrastructureUtilities.cs

```diff
@@ -0,0 +1,185 @@
+﻿using Cmf.CustomerPortal.BusinessObjects;
+using Cmf.CustomerPortal.Orchestration.CustomerEnvironmentManagement.InputObjects;
+using Cmf.LightBusinessObjects.Infrastructure;
+using Cmf.LightBusinessObjects.Infrastructure.Errors;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Cmf.CustomerPortal.Sdk.Common.Handlers
+{
+    public class InfrastructureUtilities : Utilities
```

Comment:
suggestion: I would have the Utilities live in the Services namespace (and Services folder as well) and maybe change this name to InfrastructureServices since this is more than a utility